stwcs/updatewcs/updatehdr.py

```diff
@@ -0,0 +1,795 @@
+"""
+
+:Authors: Warren Hack, Mihai Cara
+
+:License: :doc:`LICENSE`
```

Comment:
That's also not conventional. The practice we are following is to add the license information at the top of every module, though I see this package is missing the license information. That's for a separate PR. The current reference will not work.

Reply:
Removed and replaced with a more generic module level docstring in 0c497f5.